python/lsst/ts/adamSensors/mockModel.py

```diff
@@ -0,0 +1,89 @@
+from pymodbus.client.sync import ModbusTcpClient as ModbusClient
+from pymodbus.exceptions import ConnectionException, ModbusIOException
+
+
+class AdamModel:
+    """
+    Class that reads sensor voltage(s) through an ADAM 6024 device
+
+        Parameters
+        ----------
+        ip : string
+            the IP address of the ADAM 6024 controller
+        port : int
+            the port number for the ADAM 6024 controller
+
+        Attributes
+        ----------
+        client : ModbusClient
+            the pymodbus object representing the ADAM 6024
+    """
+
+    def __init__(self, log, simulation_mode=False):
+        self.client = None
+        self.clientip = None
+        self.clientport = None
+        self.log = log
+
+        self.range_size = 20
+        self.range_start = -10  # zero point offset for the ADAM device
+        self.simulation_mode = simulation_mode
+
+    def connect(self, ip, port):
+        self.clientip = ip
+        self.clientport = port
+        if self.simulation_mode:
+            self.client = None
```

Comment:
Would be nice to have a simulator (e.g. `ModbusClientSim`). Consider doing this in the future.